library/src/Select/index.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { SelectContext } from '../common/context';
+import { ISelectContext } from '../types';
+import styles from './styles.css';
+
+const SelectButton = ({
+  id,
+  children,
+}: {
+  id: string;
+  children: React.ReactNode;
+}) => {
+  const { toggleSelect } = React.useContext<ISelectContext>(
```

Comment:
Curious: Is there any difference in using a type instead of an interface here: `React.useContext(SelectContext) as MySelectContextType`

Reply:
Not sure but I would say there is: declaring the type with angle brackets makes it 100% have to conform to the type, when you use the `as` keyword you are casting it into a certain type that might be right or not.

Reply:
Gotcha. Thanks for the clarification!